### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Combinatorics/SimpleGraph/Connectivity/Subgraph.lean [hunk 1]
```diff
@@ -148,8 +147,9 @@ lemma preconnected_iff_forall_exists_walk_subgraph (H : G.Subgraph) :
       (p.toSubgraph_connected ⟨_, p.start_mem_verts_toSubgraph⟩ ⟨_, p.end_mem_verts_toSubgraph⟩)
 
 lemma connected_iff_forall_exists_walk_subgraph (H : G.Subgraph) :
-    H.Connected ↔ H.verts.Nonempty ∧ ∀ {u v}, u ∈ H.verts → v ∈ H.verts →
```
Comment (ericrbg) [inline Mathlib/Combinatorics/SimpleGraph/Connectivity/Subgraph.lean:151]: I kind of preferred the old one, but I think often I don't stick to mathlib style, so i'll let someone else decide

**Review by ericrbg — APPROVED** [review]
as usual someone else should check through it, but lgtm! thanks so much for tidying :)

maintainer merge

File: Mathlib/AlgebraicGeometry/Morphisms/QuasiCompact.lean [hunk 2]
```diff
@@ -181,7 +181,7 @@ theorem QuasiCompact.affineProperty_isLocal : (QuasiCompact.affineProperty : _).
       topIsAffineOpen _]
 #align algebraic_geometry.quasi_compact.affine_property_is_local AlgebraicGeometry.QuasiCompact.affineProperty_isLocal
 
-theorem QuasiCompact.affine_openCover_tFAE {X Y : Scheme.{u}} (f : X ⟶ Y) :
+theorem QuasiCompact.affine_openCover_tfae {X Y : Scheme.{u}} (f : X ⟶ Y) :
```
Comment (riccardobrasca) [inline Mathlib/AlgebraicGeometry/Morphisms/QuasiCompact.lean:184]: I see that we use both `tfae` and `TFAE`... do you know if there is an "official" decision about this?

File: Mathlib/CategoryTheory/Abelian/Pseudoelements.lean [hunk 3]
```diff
@@ -460,7 +460,7 @@ variable [Limits.HasPullbacks C]
     that `f p = g q`, then there is some `s : pullback f g` such that `fst s = p` and `snd s = q`.
 
     Remark: Borceux claims that `s` is unique, but this is false. See
-    `Counterexamples/Pseudoelement` for details. -/
+    `Counterexamples/Pseudoelement.lean` for details. -/
```
Comment (riccardobrasca) [inline Mathlib/CategoryTheory/Abelian/Pseudoelements.lean:463]: There is no way to make this clickable in the doc, right?

**Review by riccardobrasca — APPROVED** [review]
Just two very small comments, otherwise LGTM, thanks!

bors d+

File: Mathlib/AlgebraicGeometry/Morphisms/QuasiCompact.lean (hunk 2 above)
Comment (Ruben-VandeVelde) [reply Mathlib/AlgebraicGeometry/Morphisms/QuasiCompact.lean:184]: Zulip seems to lean towards `tfae` at the moment, so I'll leave it for now

**Comment by Ruben-VandeVelde** [comment]
bors merge

File: Mathlib/CategoryTheory/Abelian/Pseudoelements.lean (hunk 3 above)
Comment (Ruben-VandeVelde) [reply Mathlib/CategoryTheory/Abelian/Pseudoelements.lean:463]: This works now: https://github.com/leanprover/doc-gen4/pull/151

**Pull request closed without merging**